website_customer_also_purchased/static/src/less/fonts/slick.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Are this fonts necessary to include?
can you test if this can be replaced by fa icons, this in order to avoid the whole inclusion of something that for sure already exists.